src/components/site/SummaryList.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const SummaryList = props => {
+  const { items = [] } = props;
+  return (
+    <React.Fragment>
+      {items.map(item => (
+        <div  key={item.key} >
```

Comment:
this div can be removed, and you can move the `key={item.key}` to the div below "dg_summary-list__item"

Reply:
done